src/h5web/toolbar/controls/ToggleBtn.tsx

```diff
@@ -12,7 +13,8 @@ interface Props {
 }
 
 function ToggleBtn(props: Props): ReactElement {
-  const { label, icon: Icon, iconOnly, value, onChange, disabled } = props;
+  const { className, label, iconOnly, value, onChange, disabled } = props;
+  const { icon: Icon } = props;
```

Comment:
A workaround to avoid multiple line breaks, I presume ?

Reply:
yes 😄 -- I hesitated...